client/components/checklist/style.scss

```
@@ -57,6 +57,7 @@ $chevron-animation: transform 0.15s cubic-bezier( 0.175, 0.885, 0.32, 1.275 );
 		border: 2px solid var( --color-neutral-5 );
 		border-radius: 16px;
 		background: var( --color-surface );
+		z-index: 2;
```

So I think we need to add something like:

```suggestion
		z-index: z-index( 'root', '.checklist__task-icon' );;
```

In place of the value and then add a new reference here:

https://github.com/Automattic/wp-calypso/blob/821bc3648edc8aafb79c35ee5548e87deaef2696/client/assets/stylesheets/shared/functions/_z-index.scss#L89

Like so:

```
'.checklist__task-icon': 2,
```

This is providing `.checklist__task-icon` has no other parents with a `z-index` set.